src/Qdrant.Client/Grpc/PointsSelector.cs

```diff
@@ -8,12 +8,22 @@ public partial class PointsSelector
 	/// <summary>
 	/// Implicitly converts a <see cref="Guid" /> ID to a new instance of <see cref="PointsSelector"/>
 	/// </summary>
-	public static implicit operator PointsSelector(Guid id)
-		=> new() { Points = new PointsIdsList { Ids = { new PointId { Uuid = id.ToString() } } } };
+	public static implicit operator PointsSelector(Guid id) => new Guid[] { id };
 
 	/// <summary>
 	/// Implicitly converts a <see cref="ulong" /> ID to a new instance of <see cref="PointsSelector"/>
 	/// </summary>
-	public static implicit operator PointsSelector(ulong id)
-		=> new() { Points = new PointsIdsList { Ids = { new PointId { Num = id } } } };
+	public static implicit operator PointsSelector(ulong id) => new ulong[] { id };
+
+	/// <summary>
+	/// Implicitly converts an array of <see cref="ulong" /> ID to a new instance of <see cref="PointsSelector"/>
+	/// </summary>
+	public static implicit operator PointsSelector(ulong[] ids) =>
+		new() { Points = new PointsIdsList { Ids = { ids.Select(id => (PointId)id).ToArray() } } };
+
+	/// <summary>
+	/// Implicitly converts an array of <see cref="Guid" /> ID to a new instance of <see cref="PointsSelector"/>
+	/// </summary>
+	public static implicit operator PointsSelector(Guid[] ids) =>
+		new() { Points = new PointsIdsList { Ids = { ids.Select(id => (PointId)id).ToArray() } } };
```

Comment:
`.ToArray()` is unneccessary as collection initializer for `RepeatedField<T>` will enumerate the `Enumerable<PointId>` projected by LINQ. Removing also avoids an allocation.

```suggestion
		new() { Points = new PointsIdsList { Ids = { ids.Select(id => (PointId)id) } } };
```